doc/python/px-arguments.md

```diff
@@ -189,9 +190,9 @@ import plotly.express as px
 import numpy as np
 import pandas as pd
 
-gapminder = px.data.gapminder()
-gdp = np.log(gapminder['pop'] * gapminder['gdpPercap'])  # NumPy array
-fig = px.bar(gapminder, x='year', y=gdp, color='continent', labels={'y':'log gdp'},
+df = px.data.gapminder()
+gdp = np.log(df['pop'] * gapminder['gdpPercap'])  # NumPy array
```

Comment:
gapminder -> df

Reply:
(thanks to the CI !)

Reply:
yep, just saw that one in the tests. I think it's the only one I missed though!